packages/editor/src/store/selectors.js

```diff
@@ -321,20 +323,22 @@ export function getEditedPostAttribute( state, attributeName ) {
  * Returns an attribute value of the current autosave revision for a post, or
  * null if there is no autosave for the post.
  *
+ * @deprecated since 4.9. Callers should use the `getAutosaveAttribute( post, attributeName )`
+ * 			   selector from the '@wordpress/core-data' package.
+ *
  * @param {Object} state         Global application state.
  * @param {string} attributeName Autosave attribute name.
  *
  * @return {*} Autosave attribute value.
  */
 export function getAutosaveAttribute( state, attributeName ) {
```

Comment:
I guess this selector is never used anywhere in Gutenberg? A shame then that we included it with 5.0. Per my prior comment, I don't know that we even want a selector like `getAutosaveAttribute` in `core-data`, and rather we could mimic an implementation to serve as compatibility here (supporting only those subset of properties `content`, `excerpt`, `title`).